proteuslib/tools/parameter_sweep.py

```diff
@@ -17,6 +17,7 @@
 import itertools
 import warnings
 
+from scipy.interpolate import griddata
```

Comment:
Do we already pull in `scipy` as an IDAES dependency?

Reply:
yes, `scipy` is included as an IDAES dependency